code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -17,7 +17,24 @@
 	melt_temperature = MELTPOINT_STEEL
 	origin_tech = "combat=1"
 	attack_verb = list("robusted", "battered", "staved in")
-
+	storage_slots = 14
+	max_combined_w_class = 28
+	ignore_w_class = list(
+		"obj/item/weapon/weldingtool/hugetank",
```

Comment:
If it worked as it was the only reason I can think of would be some weird inbuilt sanity in text2path on Byond's side.